Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/MarkerDetection/MarkerVisualizeCoordinateService.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Extensions.Experimental.MarkerDetection;
+using Microsoft.MixedReality.Toolkit.Extensions.Experimental.Sharing;
+using System;
+using System.Numerics;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView
+{
+    /// <summary>
+    /// A variant of marker based <see cref="ISpatialCoordinateService"/> implementation. This one tracks coordinates displayed on the screen of current mobile device.
+    /// The logic is that every time you start tracking a new coordinate is created and shown on the screen, after you stop tracking that coordinates location is no longer updated with the device.
+    /// </summary>
+    public class MarkerVisualizeCoordinateService : SpatialCoordinateServiceBase<int>
```

Comment:
>MarkerVisualizeCoordinateService [](start = 17, length = 32)

nit: may be worth naming MarkerVisualizer* compared to MarkerVisualize*

Reply:
Yep, will update.

---
In reply to: [270508873](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3732#discussion_r270508873) [](ancestors = 270508873)